telemetry_core/src/main/java/com/newrelic/telemetry/metrics/json/MetricBatchJsonTelemetryBlockWriter.java

```diff
@@ -27,13 +29,21 @@ public void appendTelemetryJson(MetricBatch batch, StringBuilder builder) {
     builder.append("\"metrics\":").append("[");
     Collection<Metric> metrics = batch.getTelemetry();
 
+    AtomicInteger filteredCount = new AtomicInteger();
     builder.append(
         metrics
             .stream()
             .filter(this::isValid)
             .map(this::toJsonString)
+            .peek(x -> filteredCount.getAndIncrement())
             .collect(Collectors.joining(",")));
 
+    if (filteredCount.get() > 0) {
```

Comment:
this looks like we're counting the number that got retained, rather than filtered out. am I crazy?

Reply:
Derp, you're right.  Lemme apply mathematics.

Reply:
I mean, I assume someone tested it....